terra_layer/models.py

```diff
@@ -107,6 +150,9 @@ def save(self, **kwargs):
         if self.group:
             cache.delete(get_layer_group_cache_key(self.group.view))
 
+    def __str__(self):
+        return f"Layer<{self.id}> - {self.name}"
```

Comment:
I'd prefer to use parenthesis here, to keep consistency with Django default behavior which is

```python
    def __repr__(self):
        return '<%s: %s>' % (self.__class__.__name__, self)

    def __str__(self):
        return '%s object (%s)' % (self.__class__.__name__, self.pk)

Reply:
You mean the `%` operator ? If we were in django code i'll agree, but here it's our code. I don't see the point to respect an old convention.

Reply:
No I mean the `<pk>` instead of `(pk)`

Reply:
Okkkkk ! So I agree !

Reply:
done